dorm/dorm/Presenter/RoomManager/RoomDetail/SeparatedRoomView.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  SeparatedRoomView.swift
+//  dorm
+//
+//  Created by Minkyeong Ko on 2022/07/20.
+//
+
+import UIKit
+
+final class SeparatedRoomView: UIView {
+
+    private var studentState = ["person", "person.fill"]
+
+    private var student: Student
+    private var roomType: Int
+
+    lazy private var horizontalDivider: UIView = {
+        let divider = UIView()
+        divider.backgroundColor = .gray
+        return divider
+    }()
+
+    lazy private var verticalDivider: UIView = {
+        let divider = UIView()
+        divider.backgroundColor = .gray
+        return divider
+    }()
+
+    lazy private var roomNumberLabel: UILabel = {
+        let roomNumber = UILabel()
+        roomNumber.text = String(student.roomNumber)
+        roomNumber.font = UIFont.boldSystemFont(ofSize: 28)
+        return roomNumber
+    }()
+
+    lazy private var studentInfo: UIStackView = {
+        let studentName = UILabel()
+        studentName.text = student.name
+        studentName.font = UIFont.boldSystemFont(ofSize: 22)
+
+        let studentNationality = UILabel()
+        studentNationality.text = "Temp Country"
+        studentNationality.font = UIFont.systemFont(ofSize: 13, weight: .semibold)
+        studentNationality.textColor = .gray
+
+        let infoStack = UIStackView(arrangedSubviews: [studentName, studentNationality])
+        infoStack.axis = .vertical
+        infoStack.spacing = 10
+        return infoStack
+    }()
+
+    lazy private var detailTextView: UITextView = {
+        let textView = UITextView()
+        textView.text = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+        textView.font = UIFont.systemFont(ofSize: 90, weight: .semibold)
+        textView.isEditable = false
+        return textView
+    }()
+
+    lazy private var editButton: UIButton = {
+        let editBtn = UIButton(type: UIButton.ButtonType.system)
+        editBtn.setTitle("Edit", for: UIControl.State.normal)
+        editBtn.setTitleColor(UIColor.postechRed, for: .normal)
+        return editBtn
+    }()
+
+    init(curStudent: Student, roomType: Int) {
+        self.student = curStudent
+        self.roomType = roomType
+        super.init(frame: .zero)
+        setUpViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("Don't need coder initializer")
+    }
+
+    private func setUpViews() {
+        setUpRoomNumberLabel()
+        setUpStudentInfo()
+        setUpTextView()
+
+        if roomType != 1 {
```

Comment:
```suggestion
        guard roomType == 1 else { return }
```

guard 문을쓴다면 아래의 인덴트도 사라지고, 빠르게 리턴할 수 있지 않을까합니다 ~

Reply:
코드도 더 깔끔해지겠네요..!! 바로 적용해 보도록 하겠습니다 감사합니다!